resources/s3-buckets.go

```diff
@@ -22,9 +24,12 @@ func ListS3Buckets(s *session.Session) ([]Resource, error) {
 
 	resources := make([]Resource, 0)
 	for _, name := range buckets {
+		tags, _ := retrieveTags(svc, name)
```

Comment:
I'm ignoring the error, as the bucket might simply not be tagged and then we get a lot of 404s. I could also check what kind of error it is, but returning empty tag list instead of nil seems to work

Reply:
I think we should check the error, because if someone filters based on Tags and the Tags are missing it might delete a resource which is supposed get filtered.

Reply:
Not sure what to do if there is an error. Fail everything just because some bucket is not tagged? Skip the bucket just because it's not tagged?...

Reply:
I see at least two possible errors here: 1: There are no tags. 2: There is a network issue.

Especially the second one is interesting, because if it is some kind of network hiccup the resource does not get the tags in the properties and *aws-nuke* would filter it, besides it has tags. In this case it would get deleted which would be very unfortunate for an user.